elftools/dwarf/die.py

```diff
@@ -142,6 +167,40 @@ def set_parent(self, die):
 
     #------ PRIVATE ------#
 
+    def _search_for_ancestors(self):
+        """ Search for our parent by, starting with the CU top die,
+            setting the _parent link of each child and then iterating on
+            the child which was closest to object DIE offset without over,
+            whithan to the object DIE offset, which is an ancestor.
```

Comment:
This sentence does not make sense to me, maybe reword or split the sentence in two parts.

Reply:
yes, that was trying to stuff everything into a sentence and it got lost.